src/core/Akka.Streams/Implementation/Fusing/Ops.cs

```diff
@@ -3897,16 +3904,12 @@ async Task ProcessTask()
                         }
                     }
 
-#pragma warning disable CS4014
-                    ProcessTask();
-#pragma warning restore CS4014
+                    _ = ProcessTask();
                 }
             }
 
             public override void OnDownstreamFinish(Exception cause)
             {
-                _completionCts.Cancel();
-                _completionCts.Dispose();
```

Comment:
Moved to `PostStop()`. AFAIK we will call it after `CompleteStage()` in interpreter, plz correct if I'm wrong.

Reply:
I believe that's the case.